Ports/.port_include.sh

```diff
@@ -66,6 +66,12 @@ function run_make_install() {
     run_command make $INSTALLOPTS install "$@"
 }
 
+function run_send_to_file() {
+    echo "+ rewrite '$1'"
+	(cd "$PORT_DIR" && echo "$2" > "$1")
```

Comment:
I think in your last patch, you were appending, weren't you? If so then this is now doing something different: overwriting. Perhaps I'm mistaken, but if not, you want >> instead of > (and if you want appending, perhaps this should be run_append_to_file?)

Also, just to make it cleaner, here, instead of the echo/cd/blah dance, you should just be able to:

    run_command echo "$2" > "$1"

and it will echo for you, as well as doing the actual thing. run_patch is the only one that does an additional echo, and it does it because the shell chomps the filename so you can't see which patch it's applying.